### ISSUE TITLE
Colors in phpunit.xml will not be recognized on Windows Environment

### ISSUE THREAD

**Issue by mrodler8com** [sebastianbergmann/phpunit#3351]
| Q                   | A
| --------------------| ---------------
| PHPUnit version     | 7.4.0
| PHP version         | 7.2.11
| Installation Method | Composer
| OS                    | Windows 10 x64

### Composer info | sort
```
doctrine/instantiator              1.1.0 A small, lightweight utility to instantiate objects in PHP without invoking their constructors
myclabs/deep-copy                  1.8.1 Create deep copies (clones) of your objects
phar-io/manifest                   1.0.3 Component for reading phar.io manifest information from a PHP Archive (PHAR)
phar-io/version                    2.0.1 Library for handling version information and constraints
phpdocumentor/reflection-common    1.0.1 Common reflection classes used by phpdocumentor to reflect the code structure
phpdocumentor/reflection-docblock  4.3.0 With this component, a library can provide support for annotations via DocBlocks or otherwise retriev...
phpdocumentor/type-resolver        0.4.0
phpspec/prophecy                   1.8.0 Highly opinionated mocking framework for PHP 5.3+
phpunit/php-code-coverage          6.0.8 Library that provides collection, processing, and rendering functionality for PHP code coverage infor...
phpunit/php-file-iterator          2.0.2 FilterIterator implementation that filters files based on a list of suffixes.
phpunit/php-text-template          1.2.1 Simple template engine.
phpunit/php-timer                  2.0.0 Utility class for timing
phpunit/php-token-stream           3.0.0 Wrapper around PHP's tokenizer extension.
phpunit/phpunit                    7.4.0 The PHP Unit Testing framework.
sebastian/code-unit-reverse-lookup 1.0.1 Looks up which function or method a line of code belongs to
sebastian/comparator               3.0.2 Provides the functionality to compare PHP values for equality
sebastian/diff                     3.0.1 Diff implementation
sebastian/environment              3.1.0 Provides functionality to handle HHVM/PHP environments
sebastian/exporter                 3.1.0 Provides the functionality to export PHP variables for visualization
sebastian/global-state             2.0.0 Snapshotting of global state
sebastian/object-enumerator        3.0.3 Traverses array structures and object graphs to enumerate all referenced objects
sebastian/object-reflector         1.1.1 Allows reflection of object attributes, including inherited and non-public ones
sebastian/recursion-context        3.0.0 Provides functionality to recursively process PHP variables
sebastian/resource-operations      2.0.1 Provides a list of PHP built-in functions that operate on resources
sebastian/version                  2.0.1 Library that helps with managing the version number of Git-hosted PHP projects
theseer/tokenizer                  1.1.0 A small library for converting tokenized PHP source code into XML and potentially other formats
webmozart/assert                   1.3.0 Assertions to validate method input/output with nice error messages.
```

### phpunit.xml
``` xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/7.3/phpunit.xsd"
         bootstrap="vendor/autoload.php"
         forceCoversAnnotation="true"
         beStrictAboutCoversAnnotation="true"
         beStrictAboutOutputDuringTests="true"
         beStrictAboutTodoAnnotatedTests="true"
         colors="true"
         verbose="true">
    <testsuites>
        <testsuite name="default">
            <directory suffix="Test.php">tests</directory>
        </testsuite>
    </testsuites>

    <filter>
        <whitelist processUncoveredFilesFromWhitelist="true">
            <directory suffix=".php">src</directory>
        </whitelist>
    </filter>
</phpunit>
```

### Problem description
In the example phpunit.xml above I set colors to true. If I use the PowerShell or the old classic Commandline it will not be recognized. If I use the commandline param `--colors=always` it will work and show the colors.

**Comment by edorian** [comment]
Hey, 

do add some context as to why this is happening to the best of my undertstanding:

The xml configuration `colors=true` is equal to `--colors` or `--colors=auto` meaning PHPUnit will try to detect if the output is a termin that supports colors and act accordingly. 

`--colors=always` forces color output on terminals that don't identify themselfs as supporting colors in a way phpunit understands.

This is checked in 
https://github.com/sebastianbergmann/environment/blob/32c5cba90f7db47b1c10a777b36eccfd44ef8bd7/src/Console.php#L38-L53

so if something is wrong there with the detection this should be raised in this project I'd say?

**Comment by keradus** [comment]
not sure would it fix this concrete issue, but in general - yes, we can update environment Console:
https://github.com/sebastianbergmann/environment/pull/25

**Comment by sebastianbergmann** [comment]
Wow, long time no see, @edorian :-)

**Comment by sebastianbergmann** [comment]
@keradus Thank you!

**Comment by sebastianbergmann** [comment]
Closing this as this should be fixed by sebastianbergmann/environment#25.
